### ISSUE TITLE
Rover: add support scurve algorithm description

### ISSUE THREAD

**Issue by xianglunkai** [ArduPilot/ardupilot_wiki#5158]
https://ardupilot.org/dev/docs/rover-L1.html
The description of the current algorithm needs to be modified to maintain consistency with the current algorithm？

**Comment by rmackay9** [comment]
@xianglunkai,

Yes, indeed, I'll need to update the dev wiki navigation explanation. txs.

**Comment by Hwurzburg** [comment]
@rmackay9 still on you list?
